definitions/tools/star_align_fusion.cwl

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/env cwl-runner
+class: CommandLineTool
+cwlVersion: v1.0
+label: "STAR: align reads to transcriptome"
+baseCommand: ["/usr/local/bin/STAR"]
+requirements:
+    - class: ShellCommandRequirement
+    - class: ResourceRequirement
+      ramMin: 64000
+      coresMin: 10
+    - class: DockerRequirement
+      dockerPull: "sridnona/mgirnaseq:v3"
```

Comment:
I believe I made a similar comment on the other PR, but it seems like this should be promoted to an official `mgibio` image for inclusion here.

Reply:
changed`sridnona/mgirnaseq:v3` to  `mgibio/star`